app/src/main/java/org/mifos/mobile/ui/fragments/UpdatePasswordFragment.kt

```diff
@@ -152,10 +152,10 @@ class UpdatePasswordFragment : BaseFragment(), TextWatcher, OnFocusChangeListene
 
     }
 
-    fun showError(message: String?) {
+    fun showError(message: Int) {
         var errorMessage = message
         if (!Network.isConnected(activity)) {
-            errorMessage = getString(R.string.no_internet_connection)
+            errorMessage = R.string.no_internet_connection
```

Comment:
revert the type to String for these functions and just handle them where they are being observed